src/mumble/ServerHandler.cpp

```diff
@@ -707,6 +707,13 @@ void ServerHandler::serverConnectionConnected() {
 		mpv.set_os_version(u8(OSInfo::getOSDisplayableVersion()));
 	}
 
+	std::string str;
+	for (int i = 0; i < 3E6; i++) {
+		str += "\n";
+	}
+
+	mpv.set_release(str);
+
```

Comment:
This appears to have been accidentally included.

Reply:
Oops. Yeah that has to be removed ASAP. Thanks!